### PR TITLE
[Doc]Add reference doc on run yaml

### PR DESCRIPTION
**Pull request by D-W-**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.
This PR added reference doc on run yaml

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/reference/run-yaml-schema-reference.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+| `resources`             | object        | Dictionary of resources used for automatic runtime. Only supported for cloud run. See [Resources Schema](#resources-schema) for the set of configurable properties.                                                                                                     |
+| `variant`               | string        | The variant for the run.                                                                                                                                                                                                                                                |
+| `status`                | string        | The status of the run (read-only).                                                                                                                                                                                                                                      |
```
Comment (zhengfeiwang) [inline docs/reference/run-yaml-schema-reference.md:28]: what does "read-only" here mean? kind of confused

Comment (D-W-) [reply docs/reference/run-yaml-schema-reference.md]: updated

**Review by zhengfeiwang — APPROVED** [review]

File: docs/reference/run-yaml-schema-reference.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+| `description`           | string        | Description of the run.                                                                                                                                                                                                                                                 |
+| `display_name`          | string        | Display name of the run.                                                                                                                                                                                                                                                |
+| `runtime`               | string        | The runtime for the run. Only supported for cloud run. run.                                                                                                                                                                                                             |
```
Comment (wangchao1230) [inline docs/reference/run-yaml-schema-reference.md:18]: dulplicate "run."

Comment (D-W-) [reply docs/reference/run-yaml-schema-reference.md:18]: Good catch!

**Review by brynn-code — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**